packages/cli/src/commands/job.ts

```diff
@@ -0,0 +1,88 @@
+import ora from 'ora';
+import { runJob, getJobById, getJobByPipeline, getJobs, getLogById } from '../service/job';
+import { ResponseParams } from '../request';
+import { JobOperation } from '../types/config';
+
+const PipelineStatus: any = {
+  0: 'creating',
+  1: 'running',
+  2: 'success',
+  3: 'fail'
+};
+
+const spinner = ora();
+
+function getListJob(data: ResponseParams) {
+  data = data.rows.map((row: any) => ({
```

Comment:
```suggestion
  data = data.rows.map((row: Record<'id' | 'pipelineId' | 'status' | 'createdAt'>) => ({
```
Declare a type will be helpful

Reply:
Yes, have done this